### DISCUSSION TITLE
Welcome to cs50_sUT Discussions!

### DISCUSSION THREAD

**Discussion by mritunjay-chandra-jha** [jhamritunjay/cs50_sUT#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BKRunner** [comment]
Hello!  Hoping "3rd time's a charm".  I got stuck on the recursion assignment last time, but am determined to keep trying.
